_source/_includes/metric-shipping/telegraf-setup-mac.md

```diff
@@ -0,0 +1,4 @@
+1. Navigate to the [Influxdata portal](https://portal.influxdata.com/downloads/).
+2. Select the latest version of Telegraf in the **Version** menu.
+3. Select **MacOS (via Homebrew)** in the **Platform** menu.
+3. Run `brew install telegraf`.
```

Comment:
Only this line is relevant ( unless we tell him to download a specific version - which we dont )
 Run `brew install telegraf`.